theano/gpuarray/dnn.py

```diff
@@ -3783,7 +3783,7 @@ def _square(a):
     with inherit_stack_trace(node.outputs):
         ret = GpuDnnReduction(scal,
                               node.op.axis,
-                              node.op.acc_dtype,
+                              node.op._acc_dtype(node.inputs[0].dtype),
```

Comment:
I'm not convinced of this change. What if the user specified a different acc_dtype?

Reply:
This is superseded by the following commit.